flaml/automl.py

```diff
@@ -507,7 +507,11 @@ def custom_metric(
                 True - retrain only after search finishes; False - no retraining;
                 'budget' - do best effort to retrain without violating the time
                 budget.
-            split_type: str, default="auto" | the data split type.
+            split_type: str or splitter object, default="auto" | the data split type.
```

Comment:
Please update the docstr for split_type in `__init__()` and `retrain_from_log()`

Reply:
OK, also in `fit()`.